src/pages/VenuePage/TemplateWrapper.tsx

```diff
@@ -144,18 +143,20 @@ export const TemplateWrapper: React.FC<TemplateWrapperProps> = ({ venue }) => {
 
   // @debt remove backButton from Navbar
   return (
-    <WorldUsersProvider venueId={venue.id}>
-      <RelatedVenuesProvider venue={venue}>
-        <ReactionsProvider venueId={venue.id}>
-          <WithNavigationBar hasBackButton={hasBackButton}>
-            <AnnouncementMessage isAnnouncementUserView />
-
-            <Suspense fallback={<LoadingPage />}>{template}</Suspense>
-
-            {shouldShowChat && <ChatSidebar venue={venue} />}
-          </WithNavigationBar>
-        </ReactionsProvider>
-      </RelatedVenuesProvider>
-    </WorldUsersProvider>
+    <RelatedVenuesProvider venueId={venue.id}>
```

Comment:
It should be removed because this Provider is inside the WIthNavigationBar

Reply:
done 7b5d338